tensorflow/__init__.py

```diff
@@ -31,10 +31,10 @@
 del division
 del print_function
 
-# These symbols appear because we import the python package which
-# in turn imports from tensorflow.core and tensorflow.python. They
-# must come from this module. So python adds these symbols for the
-# resolution to succeed.
+# These symbols appear to us because we import all the python packages which
```

Comment:
"to us" is irrelevant.

"all" "packages" is invalid. There is only 1 python package.